### ISSUE TITLE
Encoding marginal Andemta annotations

### ISSUE THREAD

**Issue by DenisNosnitsin1970** [SChAth/ScAthiop#2357]
I wonder how I have to deal with Amharic commentaries in the margins, like in Abbadie 156 `https://gallica.bnf.fr/ark:/12148/btv1b100873516/f25.item.r=Abbadie%20156.zoom`. I cannot prove that it is a stable text copied later, actually it is believed that it is not. Amharic andemta it is supposed to be casual and reflects oral tradition). In this specific case I cannot say exactly if it is the main hand or not; the duplicates are of insufficient quality (more probabry it is a later hand). How shoud I encode this Andemta? As a note in extras (that the text is accompanied by... - this would be my preference)? Should I reflect the presence of Andemta anyhow else? Thanks in advance for suggestions.

**Comment by eu-genia** [comment]
Yes, I would encode the Amharic commentary as glosses in the margins `<desc type="Gloss"></desc>`, they do not seem to be an original project

**Comment by CarstenHoffmannMarburg** [comment]
It might be difficult to identify amharic glosses commenting on bible texts as stable texts in itself. These commentaries may have a paraphrasing character very often and might be very similar to one another unless they probably originated individually. I think it is very likely that these are ad hoc additions, that do not form a stable text, when put together. Judgements on the similarity of hands are as well tricky because glosses are mostly very small in size and it is hardly possible to discern reliable characteristic features. One could have better finding when analysing the inks more precisely and more widely.
